utils/io.go

```diff
@@ -15,15 +15,16 @@ func WriteKeyToPemFile(fileName string, keyPurpose string, key []byte) error {
 		return err
 	}
 
+	defer f.Close()
 	block := &pem.Block{
 		Type:  keyPurpose,
 		Bytes: key,
 	}
 	if err := pem.Encode(f, block); err != nil {
 		return err
 	}
-	f.Close()
-	return nil
+
+	return f.Chmod(os.FileMode(0600))
```

Comment:
is there no standard constant defined for this?

Reply:
There will be million combinations if they go with constants ;)

Reply:
haha but there are bit masks that can be used right?

Reply:
not critical ;)

Reply:
If you mean by using the standard perm 0777 and masking it. I think we will lose readability there. Would prefer keeping it as `0600`